frontendR2.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
```

Comment:
what does frontendR2 mean?

Reply:
I saved the frontend file in my local folder with a different name so I wouldn't be changing my main. Although I guess it doesn't matter because I could always pull my old file from Github